### PR TITLE
CIV-6601 claimant intent pi accept

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6601

### Change description ###
Claimant Intent – Pay in instalments + Accept Payment Method

### URL ###
http://localhost:3001/case/1645882162449409/claimant-response/accept-payment-method

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — APPROVED** [review]

File: src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import {NextFunction, Request, Response, Router} from 'express';
+import {
+  CLAIMANT_RESPONSE_ACCEPT_REPAYMENT_PLAN_URL,
+  CLAIMANT_RESPONSE_TASK_LIST_URL,
+} from '../../urls';
+import {GenericForm} from '../../../common/form/models/genericForm';
```
Comment (portizdev) [inline src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts:6]: Can you shorten these imports pls

File: src/test/unit/routes/features/claimantResponse/acceptRepaymentPlanController.test.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {app} from '../../../../../main/app';
 import {mockCivilClaim, mockRedisFailure} from '../../../../utils/mockDraftStore';
```
Comment (portizdev) [inline src/test/unit/routes/features/claimantResponse/acceptRepaymentPlanController.test.ts:4]: Shorter imports pls

File: src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  try {
+    const lang = req.query.lang ? req.query.lang : req.cookies.lang;
+    const claimId = req.params.id;
+    const details = await getFullAdmitSetDatePaymentDetails(claimId);
+    const claim: Claim = await getCaseDataFromStore(claimId);
+    const frequency = getRepaymentFrequency(claim);
+    repaymentPlan = {
+      paymentAmount: getPaymentAmount(claim),
+      repaymentFrequency: convertFrequencyToText(frequency, getLng(lang)),
+      firstRepaymentDate: formatDateToFullDate(new Date(getFirstRepaymentDate(claim))),
```
Comment (laperla11) [inline src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts:44]: `getFirstRepaymentDate` and `getFirstRepaymentDate` are already returning instance of Date class. do we need `new Date()` here?

**Review by laperla11 — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

File: src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts (hunk 1 above)
Comment (GorilaNaranja) [reply src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts]: done

File: src/test/unit/routes/features/claimantResponse/acceptRepaymentPlanController.test.ts (hunk 2 above)
Comment (GorilaNaranja) [reply src/test/unit/routes/features/claimantResponse/acceptRepaymentPlanController.test.ts:4]: done

File: src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts (hunk 3 above)
Comment (GorilaNaranja) [reply src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts]: Yes good point, updated!

**Review by portizdev — APPROVED** [review]

File: src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts (hunk 3 above)
Comment (laperla11) [reply src/main/routes/features/claimantResponse/acceptRepaymentPlanController.ts]: cool

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request merged**